src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -368,15 +397,14 @@
         currentActiveGroup?.classList.remove('ibexa-collapse--active-field-definitions-group');
         group.classList.add('ibexa-collapse--active-field-definitions-group');
     };
-    const removeLoadingField = () => {
-        const field = doc.querySelector('.ibexa-collapse--field-definition-loading');
+    const removeHighlight = (field) => {
```

Comment:
If you added field on position 5 and before highlight disappears (in three seconds after adding) added new field before, let's say on position 2, previous version removed highlights in order of occurences on page (so in this case, highlight from field added later was removed first), this change takes direct control on which field should have highlight removed